### ISSUE TITLE
Ability to have a 'spectator view' for a video chat host to present.

### ISSUE THREAD

**Issue by zirdum** [jakelauer/allbadcards#56]
My idea is...
When hosting a game night via some form of video chat, it would be nice if we could have a code that the chat host could sign in with that see's what's on the table view (and be able to present that screen to everyone in the chat).

**Comment by jakelauer** [comment]
This exists, you can join as a spectator as long as you use either a different browser or incognito mode.

**Comment by zirdum** [comment]
I should have been more clear but I just played around with it more and have a better understanding of how it all works. What I should have asked was, it would be nice if the person initiating the game could spectate rather than being the first player.

**Comment by jakelauer** [comment]
I see - so the request here is actually to have the owner be a spectator.

**Comment by jakelauer** [comment]
That said, it sounds like your original request would work fine as long as the owner is also playing. e.g. you could make a spectator and show that to the video chat. This works as long as the owner is also a player.

**Comment by zirdum** [comment]
> I see - so the request here is actually to have the owner be a spectator.

Correct. It would be a bit more streamlined if the device creating the game (the owner as you say) could just chose to spectate if they wanted. My workflow would be: Create the game on my host device and set it to spectate (along with zoom/Google Meet/etc). Present that spectate view to everyone via the video chat app. Allow players to join in with their client devices (including myself).
